flink-connector-mysql-cdc/src/test/resources/file/debezium-data-schema-exclude.txt

```diff
@@ -0,0 +1,2 @@
+{"before":null,"after":{"id":1,"tiny_c":127,"tiny_un_c":255,"small_c":32767,"small_un_c":65535,"int_c":2147483647,"int_un_c":4294967295,"int11_c":2147483647,"big_c":9223372036854775807,"varchar_c":"Hello World","char_c":"abc","float_c":123.10199737548828,"double_c":404.4443,"decimal_c":"EtaH","numeric_c":"AVo=","boolean_c":1,"date_c":18460,"time_c":64822000000,"datetime3_c":1595008822123,"datetime6_c":1595008822123456,"timestamp_c":"2020-07-17T18:00:22Z","file_uuid":"ZRrtCDkPSJOy8TaSPnt0AA=="},"op":"r","transaction":null}
```

Comment:
Nice change, could we make the format more readable as following, we can then use `JSONObject.parseObject("debezium-data-schema-exclude.json").getJSONObject("expect_line_1")`
```json
{
    "expect_line_1": {
        "before": null,
        "after": {
            "id": 1
        },
        "op": "r",
        "transaction": null
    },
    "expect_line_2": {
        "before": {
            "id": 1
        },
        "after": {
            "id": 1
        },
        "op": "u",
        "transaction": null
    }
}
```
And you may need to add the  `flink-connector-mysql-cdc/src/test/resources/file/*.json` to line 187 of `pom.xml` files to checkstyle.